src/entities/abTrail/ui/AbCanvas.tsx

```diff
@@ -117,8 +119,12 @@ const AbCanvas: FC<Props> = props => {
     }
   };
 
-  const isOverNode = (pointToCheck: Point, nodeIndexToCheck: number) => {
-    const node = findNodeByPoint(pointToCheck);
+  const isOverNode = (
+    pointToCheck: Point,
+    nodeIndexToCheck: number,
+    radiusMultiplier?: number,
```

Comment:
radiusMultiplier cannot be parameter of this function, because it breaks the goal of the function.
Please, offer other approach